collect_app/src/main/java/org/odk/collect/android/notifications/NotificationManagerNotifier.java

```diff
@@ -84,11 +84,10 @@ public void onUpdatesDownloaded(HashMap<ServerFormDetails, String> result) {
     @Override
     public void onSyncFailure(FormApiException exception) {
         Intent intent = new Intent(application, FillBlankFormActivity.class);
-        intent.putExtra(FillBlankFormActivity.EXTRA_AUTH_REQUIRED, true);
 
-//        if (exception.getType() == FormApiException.Type.AUTH_REQUIRED) {
-//
-//        }
+        if (exception.getType() == FormApiException.Type.AUTH_REQUIRED) {
```

Comment:
Ah, I see. Given this, I'll merge #3986 so you can clear out those commits here and not deal with merge conflicts.